set.mm

```diff
@@ -250868,22 +250876,13 @@ such that every prime ideal contains a prime element (this is a
     $( The scalar injection is a ring homomorphism.  (Contributed by Mario
        Carneiro, 8-Mar-2015.) $)
     asclrhm $p |- ( W e. AssAlg -> A e. ( F RingHom W ) ) $=
```

Comment:
A "Proof shortened by" tag should be added here (and the old proof shoulds be kept as OLD theorem)

Reply:
In the case of asclrhm, all that was changed was replacing [step 54](https://us.metamath.org/mpeuni/asclrhm.html) with ascldimul (+ 3expb)

Maybe ascldimulOLD (from asclrhm) should be added?

Reply:
It seems the unresolved issue here blocks the finishing of this PR for many days now.  Are you satisfied with this answer @avekens ?

Reply:
I still do not understand why no "Proof shortened by" tag should be added for ~asclrhm, and the old proof should be kept as theorem asclrhmOLD (according to our rules/conventions), because the proof changed significantly.

And the part of the proof replaced by ascldimul is an alternate proof for ascldimul (and should be kept as (ascldimulOLD)? If there is nothing interesting in this proof, it needs not be kept.

Reply:
The proof is pretty much exactly the same, analogous to extracting a subproof (which here is ascldimul) so that's why no old version was made. Indeed if 3expb wasn't needed the auto minimizer would probably have used ascldimul.

But I will add the ascldimulOLD for comparison